### DISCUSSION TITLE
Issue with Bootstrap Code Displaying in Arabic Instead of English

### DISCUSSION THREAD

**Discussion by 3101ajeet** [twbs/bootstrap#38792] [Q&A]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Hello Bootstrap Team,

I am encountering an issue where the Bootstrap code is displaying in Arabic instead of English. I have copied a starter template from Bootstrap, but the text and labels in the code appear in Arabic by default. I have made sure to set the language attribute to "en" in the HTML file, but the issue persists.

I have also checked my browser settings to ensure that the default language is set to English and disabled any translation extensions or add-ons that may interfere with the language settings.

I would like to use the Bootstrap code in English for my project. Is there any additional step or configuration that I am missing to ensure the code is displayed in English?

I appreciate your assistance in resolving this issue. Please let me know if there are any suggestions or solutions to fix this problem.

Thank you for your time and support.

Best regards,
Ajeet Kumar
ajitsingh.hr01@gmail.com
[starter template.docx](https://github.com/twbs/bootstrap/files/11791300/starter.template.docx)

### Reduced test cases

do not have

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

v5.3

**Comment by julien-deramond** [comment]
Hey @3101ajeet 

I've converted your issue to a discussion since it's probably rather a usage problem than an actual issue in Bootstrap.

By default, Bootstrap is in LTR (e.g. https://getbootstrap.com/docs/5.3/getting-started/introduction/).
RTL is handled separately and requires extra steps: https://getbootstrap.com/docs/5.3/getting-started/rtl/#required-html.

Based on the few elements I have in your use case, I'd say that you can:
- Remove dir="rtl" on the `<html>` element
- Set `lang="en"`, on the `<html>` element.
- Load `bootstrap.min.css` instead of `bootstrap.rtl.min.css`
